index.html

```diff
@@ -5,15 +5,91 @@
+        <input type="checkbox" id="toggle"/>
+        <div class="nav-a-container">
+            <a class="nav-a-first"><strong>Karma</strong></a>
+            <a class="nav-a">How it Works</a>
+            <a class="nav-a">Store</a>
+            <a class="nav-a">Blog</a>
+            <a class="nav-a">Help</a>
+            <a class="nav-a">Login</a>
+        </div>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+    </nav>
+
+    <article>
+        <header> 
+            <div class="header-div">
+                <div class="header-image">  </div>
+                
+                <!-- <img class="header-image" src="img\first-background.jpg" alt="An ipad and karma Wifi device on a table with a person in a white shirt looking at these devices"> -->
+                    <div class="inner-header-div">
+                        <h1>Introducing Karma</h1>
+                        <p>Bring WiFi with you, everywhere you go.</p>
+                        <button type="button">Learn More</button>
+                    </div>                                                                                                                                           
+            </div>
+        </header>
+    </article>
+    
+    <main>
+
+        <h2>Everyone needs a little Karma</h2>
+
+        <div class="main-img-container">
+            <figure class="main-img">
+                <img  src="img\icon-devices.svg" alt="Logo in orange representing a tablet, phone and a desktop">
+                <figcaption>Internet for all devices</figcaption>
+            </figure>
+
+            <figure class="main-img" >
+                <img src="img\icon-coffee.svg" alt="Logo in orange of a coffee mug with steam coming out of the top">
+                <figcaption>Boost your productivity</figcaption>
+            </figure>
+            <figure class="main-img">
+                <img src="img\icon-refill.svg" alt="Logo in orange of a fuel refilling station">
+                <figcaption>Pay as you go</figcaption>
+            </figure>
+        </div>
+        
+        
+        <div class="extended-box">
+            <img class="extended-box-img" src="additional\feature-img.png" alt="Logo in orange representing a tablet, phone and a desktop">
+            
+            <div class="text-div">
+            <p class="extended-box-p"> Wherever I am, I just don't worry about my connection anymore!</p>
+            <a class="extended-box-a">Get Karma today</a>
+            </div>
+
+        </div>
+        
+    </main>
 
 </body>
+
+    <footer>
+
+        <p>Join us on</p>
+        
+        <div class="footer-container">
+            
+            <i class="fab fa-twitter"></i>
+            <i class="fab fa-facebook-f"></i>
+            <i class="fab fa-instagram"></i>
+
+        </div>
+    </footer>
```

Comment:
the footer should be inside the body tag (anything that forms part of the document structure goes in body). Browsers are sometimes a bit too helpful - most will fix this when you open the page! Great for a user but a bit annoying when you are developing :)